File: String/Character-Replace/character_replace.py
def swap_a_p(s):
    result = ""
    for ch in s:
        if ch == 'a':
            result += 'p'
        elif ch == 'p':
            result += 'a'
        else:
            result += ch
    print(result)

# Example usage
swap_a_p("apple")

File: String/Character-Replace/CharacterReplace.js
function swapAandP(str) {
  let result = ''
  for (let i = 0; i < str.length; i++) {
    if (str[i] === 'a') {
      result += 'p'
    } else if (str[i] === 'p') {
      result += 'a'
    } else {
      result += str[i]
    }
  }
  console.log(result)
}

// Example usage
swapAandP('apple')
